jbmc/regression/janalyzer-taint/taint-aliasing1/test.desc

```diff
@@ -1,5 +1,5 @@
 CORE
-aliasing1.class
+aliasing1
```

Comment:
:no_entry_sign: The commit message does not match with this change
` Make janalyzer use lazy goto model initialization`